cmd/update.go

```diff
@@ -129,6 +136,7 @@ func newUpdateCmd() *updateCmd {
 	}
 
 	root.cmd = cmd
+	root.cmd.Flags().BoolVarP(&root.opts.dryRun, "dry-run", "", false, "Only show status, don't prompt for update")
```

Comment:
I would mention, that dry run mode exits with non `0` exit code, if updates have been found and `0` otherwise.

```suggestion
	root.cmd.Flags().BoolVarP(&root.opts.dryRun, "dry-run", "", false, "Only show status, don't prompt for update (exit code is non-zero if there are updates available and 0 otherwise)")
```

Reply:
I don't think this clarification belongs here. It should be on a `man` page at most, but I don't usually see CLI flags saying what status code they return.

Reply:
If this is meant to be used in scripting I would tend to agree with @breml. It promises the user that they can rely on the exit code. But in that case I would expect a different exit code if there is an actual error.

An alternative to the flag would be a new command, e.g. `updates-available`.

Maybe we could lean on git's porcelain vs. plumbing commands: https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain.

Reply:
> But in that case I would expect a different exit code if there is an actual error.

True, if that's the case we should differentiate from "updates" vs "any other error" 

> An alternative to the flag would be a new command, e.g. `updates-available`.

Hmmm I think at this point I'm more inclined to leave it as a flag than to add a new command just for that. I do agree that we should handle the exit code thing though. I'll update that.